src/app/account/edit/page.tsx

```diff
@@ -70,22 +71,24 @@ export default function EditAccount() {
     })
 
     useEffect(() => {
-        (async () => {
-            const accountInfo = await getUserAccount()
+        ;(async () => {
```

Comment:
I haven't touch react in some time but since we are using the Nextjs App router with React server components. We should be able to lift the async function to the top level and let the server handle it.